src/proto/Cardano.proto

```diff
@@ -11,7 +11,7 @@ message OutPoint {
 }
 
 message TokenAmount {
-    string policy_id = 1;
+    string policy_id = 1; // as hex string (28x2 digits)
```

Comment:
what is policy id and how to get it?

Reply:
Policy ID is the ID of the token, somewhat analogue to the contract ID for ERC20 tokens.  However, in Cardano a policy can define multiple tokens, so a token is identified by the policyID and assetName together.
E.g. https://cardanoscan.io/tokens lists policyIDs and assetNames.
PolicyID is a hash, BTW.